controllers/flagsourceconfiguration_controller.go

```diff
@@ -18,45 +18,146 @@ package controllers
 
 import (
 	"context"
+	"fmt"
+	"strings"
+	"time"
 
+	appsV1 "k8s.io/api/apps/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
 	"k8s.io/apimachinery/pkg/runtime"
 	ctrl "sigs.k8s.io/controller-runtime"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 	"sigs.k8s.io/controller-runtime/pkg/log"
+	"sigs.k8s.io/controller-runtime/pkg/predicate"
 
+	"github.com/go-logr/logr"
 	corev1alpha1 "github.com/open-feature/open-feature-operator/apis/core/v1alpha1"
```

Comment:
all these imports need to be adapted to v1alpha3, v1alpha1 and v1alpha2 versions can be present and used only in conversion webhooks

Reply:
See [here](https://github.com/open-feature/open-feature-operator/pull/344/files#r1113202711)